README.md

```diff
@@ -95,16 +95,22 @@ const OrbitDB = require('orbit-db')
 
 // Create IPFS instance
 
-// For js-ipfs >= 0.38
-const ipfs = new IPFS()
-
-// For js-ipfs < 0.38
-const ipfsOptions = {
-  EXPERIMENTAL: {
-    pubsub: true
+var ipfs;
```

Comment:
Not to split hairs here but could this be expressed as const `ipfs = initIPFSInstance()`

```javaScript
const initIPFSInstance = async () => {
  // For js-ipfs >= 0.38
  ipfs = await IPFS.create({ repo: "./path-for-js-ipfs-repo" });

  // For js-ipfs < 0.38
  const ipfsOptions = {
    EXPERIMENTAL: {
      pubsub: true
    }
  }

  ipfs = new IPFS(ipfsOptions)
}

const ipfs = initIPFSInstance();
```

Reply:
unfortunately, that gives a "declaration to const variable" error.